### PR TITLE
Remove auto admit description

### PR DESCRIPTION
**Pull request by fafan**
It's not the case for Teams anymore, so we need to change the language.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/reset-the-audio-conferencing-pin-in-teams.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Meetings can be started when an authenticated user joins using the Microsoft Tea
     
 - Automatically sending emails to users is enabled by default, and users will receive an email with their PIN when they're enabled for audio conferencing or when the PIN is reset. But if you have disabled automatically sending emails, a PIN reset email won't be sent to a user and you will have to manually send the PIN information to the user.
     
-- When a meeting starts, all of the users in the lobby will automatically join it. For example, if two participants try to join a meeting before it has been started, they will be put in the lobby and will listen to music on hold, and when the meeting organizer joins using their PIN via phone, the meeting will start and the participants in the lobby will join the meeting.
+- When a meeting starts, organizer needs to admit all PSTN users in the lobby to join the meeting. For example, if two PSTN participants try to join a meeting before it has been started, they will be put in the lobby and will listen to music on hold, and when the meeting organizer joins using their PIN via phone, the meeting will start and the organizer can use the in-meeting command (press *21) to admit all PSTN users in the lobby.
```
Comment (JohanFreelancer9) [inline Teams/reset-the-audio-conferencing-pin-in-teams.md:62]:
```suggestion
- When a meeting starts, the organizer needs to admit all PSTN users in the lobby to join the meeting. For example, if two PSTN participants try to join a meeting before it has started, they will be put in the lobby and will listen to music on hold, and when the meeting organizer joins using their PIN via phone, the meeting will start and the organizer can use the in-meeting command (press *21) to admit all PSTN users in the lobby.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @fafan, kindly commit my minor suggestions. Many thanks.

**Comment by fafan** [comment]
Hi @JohanFreelancer9 , just commit the changes. Thanks!

**Comment by fafan** [comment]
@JohanFreelancer9 Anything else needed from my side? When the change will go live?

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**